data/preservedata.rb

```diff
@@ -0,0 +1,171 @@
+require_relative '../book'
+require_relative '../person'
+require_relative '../student'
+require_relative '../teacher'
+require 'json'
+
+def save_book(book)
+  book_obj = {
```

Comment:
**[OPTIONAL]**
- [ ] I was adding a rental and then your application asked for a book title instead of showing it by numbers and asking for selecting a number :thinking: which was a requirement of your previous project :thermometer: Kindly fix this issue to make your project professional :+1: